ReleaseTooling/Sources/FirebaseManifest/Pod.swift

```diff
@@ -56,12 +56,9 @@ public struct Pod {
   }
 
   /// The Firebase pod does not support import validation with Xcode 12 because of the deprecated
-  /// ML pods not supporting the ARM Mac slice.
+  /// ML pods not supporting the ARM Mac slice. While all other pods support
+  // it, it is skipped to decrease the time it takes to stage and publish them.
```

Comment:
Could this be simplified to:
```
/// Import validation is skipped to decrease the time it takes to stage and publish the pod.
```
The ML Kit pods were removed in [Firebase 8.0](https://github.com/firebase/firebase-ios-sdk/pull/7914) and I don't think the comment is referring to the ML Model Downloader.